source/hardware/controllers/denon_mc7000.rst

```diff
@@ -277,9 +302,15 @@ This mode lets you jump a number of beats while pushing a Pad once.
    :alt: Denon MC7000 BEATJUMP Mode
    :figclass: pretty-figures
 
+| Press and hold ":hwlabel:`SHIFT`" and then push a PAD button to jump even more beats.
+| 16, 32, 64 and 128 forward on the upper row and backwards on the lower row.
```

Comment:
```suggestion
Press and hold :hwlabel:`SHIFT` and then push a pad button to jump even more beats.
16, 32, 64 and 128 forward on the upper row and backwards on the lower row.
```

Reply:
wanna keep line breaks as on the previous comment so that all possible values are not broken between lines. "pad" is now lowercase